tests/countries/test_turks_and_caicos_islands.py

```diff
@@ -0,0 +1,219 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.turks_and_caicos_islands import TurksAndCaicosIslands, TC, TCA
+from tests.common import CommonCountryTests
+
+
+class TestTC(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(TurksAndCaicosIslands, years=range(2010, 2025))
+
+    def test_country_aliases(self):
+        self.assertAliases(TurksAndCaicosIslands, TC, TCA)
+
+    def test_commonwealth_day(self):
+        name = "Commonwealth Day"
+        self.assertHolidayName(
+            name,
+            "2020-03-09",
+            "2021-03-08",
+            "2022-03-14",
+            "2023-03-13",
+            "2024-03-11",
+            "2025-03-10",
+        )
```

Comment:
```suggestion
        )
        self.assertHolidayName(name, range(1963, 2050))
```
And the same for all simlar holidays.